### DISCUSSION TITLE
Potion et objet de temps

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1985] [Suggestions fermées]
Ce serait bien je pense que les potions et objets qui avancent le temps influencent aussi notre énergie en la régènerant.

**Comment by Nargacaura** [comment]
Le concept est déjà expliqué dans l'issue #1872, et il a été refusé car ce n'est pas nécessaire au vu de la présence de popos dédiées à ça. (Je te laisse lire la discussion dans l'issue mentionnée)

**Reply by ghost** [reply]
Ah okay, oui maintenant que je vois ça je comprends mieux merci. 👌

**Comment by BastLast** [comment]
Je ferme
